### PR TITLE
#3395 Voltage values of VRRemote are now in Pu

### LINKED ISSUE #3395: [DynaFlow] VRRemote's NQ signal is computed with absolute values (in kV) whereas they are in Pu on generator side

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #3395

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

**Review by gaouizer — APPROVED** [review]

**Review by marcochiaramello — APPROVED** [review]

File: nrt/data/DynaFlow/VRRemote/VRRemote.par [hunk 1]
```diff
@@ -62,15 +62,15 @@
   </set>
   <set id="SignalNQ">
     <par type="BOOL" name="vrremote_FreezingActivated" value="true"/>
-    <par type="DOUBLE" name="vrremote_U0" value="1.05"/>
-    <par type="DOUBLE" name="vrremote_URef0" value="1.05"/>
+    <par type="DOUBLE" name="vrremote_U0Pu" value="1.05"/>
+    <par type="DOUBLE" name="vrremote_URef0Pu" value="1.05"/>
     <par type="DOUBLE" name="vrremote_Gain" value="1"/>
     <par type="DOUBLE" name="vrremote_tIntegral" value="1"/>
```
Comment (marcochiaramello) [inline nrt/data/DynaFlow/VRRemote/VRRemote.par:68]: the value here should be ailgned to waht is given by default in DFL

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo [hunk 2]
```diff
@@ -25,18 +25,18 @@ model VRRemote "Model for coordinated primary voltage regulation. This model is
     Placement(visible = true, transformation(origin = {-110, -60}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-110, -60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Interfaces.BooleanInput[NbGenMax] limUQUp(start = limUQUp0) "Whether the maximum reactive power limits are reached or not (for each generator participating in the coordinated primary voltage regulation of the considered bus)" annotation(
     Placement(visible = true, transformation(origin = {-110, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-110, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Modelica.Blocks.Interfaces.RealInput URef(start = URef0) "Regulated voltage reference in kV" annotation(
+  Modelica.Blocks.Interfaces.RealInput URefPu(start = URef0Pu) "Regulated voltage reference in per-unit" annotation(
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo:28]: in pu (base UNom)

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo [hunk 3]
```diff
@@ -25,18 +25,18 @@ model VRRemote "Model for coordinated primary voltage regulation. This model is
...
     Placement(visible = true, transformation(origin = {-110, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-110, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Modelica.Blocks.Interfaces.RealInput URegulated(start = U0) "Regulated voltage in kV" annotation(
+  Modelica.Blocks.Interfaces.RealInput URegulatedPu(start = U0Pu) "Regulated voltage in per-unit" annotation(
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo:30]: in pu (base UNom)

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo [hunk 4]
```diff
@@ -25,18 +25,18 @@ model VRRemote "Model for coordinated primary voltage regulation. This model is
...
     Placement(visible = true, transformation(origin = {110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
 
   parameter Boolean[NbGenMax] limUQDown0 = fill(true, NbGenMax) "Whether the minimum reactive power limits are initially reached or not (for each generator participating in the coordinated primary voltage regulation of the considered bus)";
   parameter Boolean[NbGenMax] limUQUp0 = fill(true, NbGenMax) "Whether the maximum reactive power limits are initially reached or not (for each generator participating in the coordinated primary voltage regulation of the considered bus)";
-  parameter Types.VoltageModule U0 "Start value of the regulated voltage in kV";
-  parameter Types.VoltageModule URef0 "Start value of the regulated voltage reference in kV";
+  parameter Types.VoltageModule U0Pu "Start value of the regulated voltage in per-unit";
+  parameter Types.VoltageModule URef0Pu "Start value of the regulated voltage reference in per-unit";
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo:39]: in pu (base Unom) for U0Pu and URef0Pu

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo (hunk 2 above)
Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo]: done

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo (hunk 4 above)
Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo]: done

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo (hunk 3 above)
Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/VRRemote.mo]: done

**Review by joyelfeghali — APPROVED** [review]

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**